src/utils/format-numbers.js

```diff
@@ -92,6 +93,14 @@ export function formatMilliseconds(
   return formatNumber(time, significantDigits, maxFractionalDigits) + 'ms';
 }
 
+export function formatSeconds(
+  time: Seconds,
```

Comment:
If I'm not mistaken, the input is in `Milliseconds` (as I see you divide by 1000 below).